src/tools/compiletest/src/header.rs

```diff
@@ -1051,7 +1075,8 @@ fn iter_header(
 
             let rest = rest.trim_start();
 
-            let CheckDirectiveResult { is_known_directive, directive_name } = check_directive(rest);
+            let CheckDirectiveResult { is_known_directive, directive_name, trailing_directive: _ } =
```

Comment:
Probably just

```suggestion
            let CheckDirectiveResult { is_known_directive, directive_name, .. } =
```

if we're not going to use the trailing directive.